web/.env.example

```
@@ -0,0 +1,6 @@
+# Environment Variables
+
+# Admin credentials for scripts documentation
+# Change these to secure values in production
+VITE_ADMIN_USERNAME=admin
+VITE_ADMIN_PASSWORD=changeme123
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_ADMIN_PASSWORD key should go before the VITE_ADMIN_USERNAME key

(UnorderedKey)